SimTracker/TrackAssociatorProducers/python/quickTrackAssociatorByHits_cfi.py

```diff
@@ -20,3 +20,6 @@
     quickTrackAssociatorByHits.associateStrip = False
     quickTrackAssociatorByHits.associatePixel = False
     quickTrackAssociatorByHits.useClusterTPAssociation = False
+
+eras.phase2_muon.toModify( quickTrackAssociatorByHits, pixelSimLinkSrc = cms.InputTag("simSiPixelDigis","Pixel") )
+eras.phase2_muon.toModify( quickTrackAssociatorByHits, stripSimLinkSrc = cms.InputTag("simSiPixelDigis","Tracker") )
```

Comment:
I believe these should be `phase2_tracker` as in `MuonAssociatorByHits_cfi.py` and `muonAssociatorByHitsNoSimHitsHelper_cfi.py` above.

(Are you running a configuration without `tpClusterProducer`? That is the only way I can see these would have any effect on top of what is done in tracker customize functions https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/SLHCUpgradeSimulations/Configuration/python/phase2TkTilted.py#L336. But yes, given this fallback mechanism https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/SimTracker/TrackAssociatorProducers/plugins/QuickTrackAssociatorByHitsProducer.cc#L166 it is good to keep these parameters in `quickTrackAssociatorByHits` and `tpClusterProducer` in synch. Maybe next time I get hit by the fallback I just remove it.)

(personally I would use `pixelSimLinkSrc = "simSiPixelDigis:Pixel"`, i.e. avoid explicit type for an existing parameter, to benefit from clone/toModify's typo checker)